src/com/javabasic/lesson6/MinimalValue.java

```diff
@@ -0,0 +1,49 @@
+package com.javabasic.lesson6;
+
+import java.util.Arrays;
+import java.util.OptionalDouble;
+
+public class MinimalValue {
+
+    public static void main(String[] args) {
+
+        double[] valuesList = {5.5, 2, 4.6};
+
+        double minimalValue = getMinimalDouble(valuesList);
+
+        getResult(minimalValue);
+
+    }
+
+    public static double getMinimalDouble(double[] valuesList) {
+        OptionalDouble minimalValue = Arrays.stream(valuesList).min();
+
+        if (minimalValue.isEmpty()) {
+            return 0;
+        }
+        else {
+            return minimalValue.getAsDouble();
+        }
+    }
+
+    public static boolean isInt(double value) {
+
+        boolean is_result;
```

Comment:
is_result is a python notation
Try to avoid mixing notations
isResult for Java